scss/application.scss

```diff
@@ -0,0 +1,7 @@
+
+
+
+@import '_sizes';
```

Comment:
Delete all this empty space
Also add empty line at end of file
You should import your equalizer at the top of yout application css file